backend/danswer/background/update.py

```diff
@@ -160,18 +161,18 @@ def create_indexing_jobs(existing_jobs: dict[int, Future | SimpleJob]) -> None:
             ongoing.add(
                 (
                     attempt.connector_credential_pair_id,
-                    attempt.embedding_model_id,
+                    attempt.search_settings_id,
                 )
             )
 
-        embedding_models = [get_current_db_embedding_model(db_session)]
-        secondary_embedding_model = get_secondary_db_embedding_model(db_session)
-        if secondary_embedding_model is not None:
-            embedding_models.append(secondary_embedding_model)
+        search_settings = [get_current_search_settings(db_session)]
```

Comment:
Feels like get_search_settings with an enum passed in, instead of two separate functions, might be cleaner.